app/_src/production/dp-config/cni.md

```diff
@@ -1,5 +1,6 @@
 ---
-title: Kuma CNI
+title: Configure the Kuma CNI
```

Comment:
Should we get rid of product names in titles? It makes it easier to reuse the sidebar between products (as we can't use `site.product_name` in the sidebar?

Reply:
@lahabana I think that we have automation on the Mesh docs side that automatically replaces "kuma" in titles/nav/URLs with "kong mesh". @fabianrbz Is that accurate?

Reply:
I believe so

Reply:
Even in the nav? Last time I checked that wasn't the case for the nav. This was making things harder to potentially add scripting to sync automatically the sidebar instead of duplicating it like we do nowadays.

Reply:
I started working on an extension for the site-generator to handle inheriting a sidebar: https://github.com/Kong/jekyll-generator-single-source/pull/7

Unfortunately I'm a bit busy atm and don't have time to complete this work.